### DISCUSSION TITLE
[7주차 알고리즘 스터디] 다이나믹 프로그래밍(DP)

### DISCUSSION THREAD

**Discussion by bigsaigon333** [fe-backpackers/js-algorithm-basecamp#69] [General]
# [7주차 알고리즘 스터디] 다이나믹 프로그래밍(DP)

## 회고 순서
체크인 : ~22:05
다른 사람 코드 참고하여 나의 코드 개선: ~22:20
자유로운 의견교환: ~22:40
개인 회고: ~22:50
팀별 공유: ~23:00

**Comment by bigsaigon333** [comment]
## 체크인

현재 기분을 10점 만점으로 표현해주세요~

**Reply by ganeodolu** [reply]
7점 : 밀린 일들이 있어서 초조한 느낌이 있습니당ㅎㅎ

**Reply by bigsaigon333** [reply]
7점: 내일 면접이 잡혀있어서 매우 긴장됩니다 :sob:

**Comment by bigsaigon333** [comment]
## 버퍼주 계획 공유

**Reply by ganeodolu** [reply]
퇴사 문제 풀어보고, 최단경로 난이도가 어려울 것으로 예상되어 조금씩 미리 봐야될 것 같습니다.

**Reply by bigsaigon333** [reply]
1. DP 실전문제를 풀이할 생각입니다.  이론정리는 오늘 거의다 해서 일단 push 할 생각이에요~
2. 퀵 정렬 부분 공부할 생각입니다!  6-4.js ~ 6-9.js 도 풀거에요 :smile: 
3. 최단 경로와 그래프 이론에 대한 동빈나 유튜브 강의 들을 계획입니다 :+1:

**Comment by bigsaigon333** [comment]
## 체크아웃

나갈 때의 기분을 표현해봐요~~

**Reply by bigsaigon333** [reply]
8점: 조촐한 알고리즘 회고 시간이었습니다. 각자 그냥 부족한 부분 학습하였습니다 :smile: 
이렇게라도 해서 알고리즘 공부할 수 있어서 다행입니다 :+1:

**Reply by ganeodolu** [reply]
8점 : 스터디 잘 버티고 있는 것 셀프칭찬과 함께 동동님 내일 화이팅입니당ㅎㅎ
